### PR TITLE
SBERDOMA-722 Role descirption card

### PR DESCRIPTION
**Pull request by MrFoxPro**
desc

### CODE REVIEW

File: apps/condo/domains/organization/components/EmployeeProfileForm/EmployeeRoleSelect.tsx [hunk 1]
```diff
@@ -15,18 +16,20 @@ export const EmployeeRoleSelect: React.FC<IEmployeeRoleSelectProps> = (props) =>
...
+        console.log('convertedOption', convertedOption)
+        console.log('hii')
         if (convertedOption) {
             const { value, label } = convertedOption
 
             return (<Select.Option key={value} value={value} title={label}>{label}</Select.Option>)
         }
     }), [employeeRoles, employee])
-
+    useEffect(()=> console.log(options), [options])
```
Comment (Dimitreee) [inline apps/condo/domains/organization/components/EmployeeProfileForm/EmployeeRoleSelect.tsx:27]: remove log

File: apps/condo/pages/employee/create.tsx [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
+import { useState } from 'react'
 
 interface IPageWithHeaderAction extends React.FC {
     headerAction?: JSX.Element
 }
 
 const CreateEmployeePage: IPageWithHeaderAction = () => {
     const intl = useIntl()
-    const PageTitleMsg = intl.formatMessage({ id:'employee.AddEmployee' })
+    const PageTitleMsg = intl.formatMessage({ id: 'employee.AddEmployee' }, {
```
Comment (Dimitreee) [inline apps/condo/pages/employee/create.tsx:19]: useless second arg

File: apps/condo/domains/organization/components/EmployeeProfileForm/EmployeeRoleSelect.tsx [hunk 3]
```diff
@@ -15,18 +16,20 @@ export const EmployeeRoleSelect: React.FC<IEmployeeRoleSelectProps> = (props) =>
     const { objs: employeeRoles, loading, error } = OrganizationEmployeeRole.useObjects({ where: { organization: { id: employeeOrganizationId } } })
     const options = useMemo(() => employeeRoles.map((employeeRole) => {
         const convertedOption = OrganizationEmployeeRole.convertGQLItemToFormSelectState(employeeRole)
-
+        console.log('convertedOption', convertedOption)
+        console.log('hii')
```
Comment (toplenboren) [inline apps/condo/domains/organization/components/EmployeeProfileForm/EmployeeRoleSelect.tsx:20]: Please remove console.logs

File: apps/condo/pages/employee/create.tsx [hunk 4]
```diff
@@ -30,8 +40,24 @@ const CreateEmployeePage: IPageWithHeaderAction = () => {
...
-                                <CreateEmployeeForm/>
+                                <CreateEmployeeForm onRoleSelect={(role) => setSelectedRole(role)} />
                             </Col>
+                            {roleTranslations.title && roleTranslations.description && <Card
+                                title={roleTranslations.title}
+                                bordered={false}
+                                style={{
+                                    width: 300,
+                                    height: 'min-content',
+                                    boxShadow: ' 0px 9px 28px 8px rgba(0, 0, 0, 0.05), 0px 6px 16px rgba(0, 0, 0, 0.08), 0px 3px 6px -4px rgba(0, 0, 0, 0.12)',
```
Comment (Dimitreee) [inline apps/condo/pages/employee/create.tsx:51]: move to const/style

File: apps/condo/domains/organization/components/EmployeeProfileForm/EmployeeRoleSelect.tsx (hunk 1 above)
Comment (toplenboren) [inline apps/condo/domains/organization/components/EmployeeProfileForm/EmployeeRoleSelect.tsx:27]: ^^

File: apps/condo/pages/employee/create.tsx [hunk 5]
```diff
@@ -30,8 +40,24 @@ const CreateEmployeePage: IPageWithHeaderAction = () => {
...
                             <Col span={10}>
-                                <CreateEmployeeForm/>
+                                <CreateEmployeeForm onRoleSelect={(role) => setSelectedRole(role)} />
                             </Col>
+                            {roleTranslations.title && roleTranslations.description && <Card
+                                title={roleTranslations.title}
+                                bordered={false}
+                                style={{
+                                    width: 300,
+                                    height: 'min-content',
```
Comment (Dimitreee) [inline apps/condo/pages/employee/create.tsx:50]: https://caniuse.com/?search=min-content

File: apps/condo/pages/employee/create.tsx [hunk 6]
```diff
@@ -30,8 +40,24 @@ const CreateEmployeePage: IPageWithHeaderAction = () => {
                                 <Typography.Title level={1} style={{ margin: 0 }}>{PageTitleMsg}</Typography.Title>
                             </Col>
                             <Col span={10}>
-                                <CreateEmployeeForm/>
+                                <CreateEmployeeForm onRoleSelect={(role) => setSelectedRole(role)} />
                             </Col>
+                            {roleTranslations.title && roleTranslations.description && <Card
+                                title={roleTranslations.title}
+                                bordered={false}
+                                style={{
```
Comment (toplenboren) [inline apps/condo/pages/employee/create.tsx:48]: I think its more performant to use styled component for this

File: apps/condo/pages/employee/create.tsx [hunk 7]
```diff
@@ -1,21 +1,31 @@
...
 interface IPageWithHeaderAction extends React.FC {
     headerAction?: JSX.Element
 }
 
 const CreateEmployeePage: IPageWithHeaderAction = () => {
     const intl = useIntl()
-    const PageTitleMsg = intl.formatMessage({ id:'employee.AddEmployee' })
+    const PageTitleMsg = intl.formatMessage({ id: 'employee.AddEmployee' }, {
+        
+    })
```
Comment (toplenboren) [inline apps/condo/pages/employee/create.tsx:21]: Is everything OK with formatting here?

File: apps/condo/domains/organization/components/EmployeeProfileForm/EmployeeRoleSelect.tsx (hunk 3 above)
Comment (pahaz) [reply apps/condo/domains/organization/components/EmployeeProfileForm/EmployeeRoleSelect.tsx:20]: Please add WIP. It’s looks like not ready for review

File: apps/condo/lang/ru.json [hunk 8]
```diff
@@ -341,6 +341,26 @@
...
+  "employee.role.description.Диспетчер": "Диспетчеры взаимодействуют с клиентом и внутренними службами УК. Диспетчеры оформляют заявки, оставляют комментарии, меняют статусы, вносят информацию по показаниям приборов учета и выдают ключи. Диспетчеры имеют доступ до информации о клиентах, начислениях и техническом персонале, но не могут ничего редактировать, добавлять или удалять.",
+  "employee.role.description.Управляющий": "Управляющий это менеджер дома: взаимодействует с клиентами, следит за качеством, сроками и выполнением заявок, контролирует финансы дома — управляет домом или группой домов. В рамках назначенных домов управляющий имеет доступ до управления заявками, управление сотрудниками, отчетов по дому, начислений и платежей.",
+  "employee.role.description.Мастер участка": "Мастер участка отвечает за выполнение и качество работ на участке. Мастер участка распределяет задачи по исполнителям и настраивает маршрутизацию.",
+  "employee.role.description.Техник": "Техник работает с задачами и выполняет работы. Техник не имеет доступа до других разделов и возможностей системы, кроме заявок.",
```
Comment (MrFoxPro) [inline apps/condo/lang/ru.json:363]: temporary solution for supressing warnings from react-intl

**Review by Dimitreee — APPROVED** [review]

File: apps/condo/domains/organization/components/EmployeeRoleSelect.tsx [hunk 9]
```diff
@@ -1,29 +1,40 @@
 import React, { useMemo } from 'react'
 import { Select, SelectProps } from 'antd'
 import { OrganizationEmployeeRole } from '@condo/domains/organization/utils/clientSchema'
+import { Loader } from '../../common/components/Loader'
```
Comment (dkoviazin) [inline apps/condo/domains/organization/components/EmployeeRoleSelect.tsx:4]:
```
import { Loader } from '@condo/domains/common/components/Loader'
```

**Review by dkoviazin — APPROVED** [review]

**Pull request merged**